index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Apple store</title>
+  <link rel="stylesheet" href="./styles/index.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <div class="main-page">
+    <div class="navbar-container">
+      <div class="navbar">
+        <div><a><img src="./img/apple-img.png" height="25" alt="apple image" /></a></div>
+        <div><a href="#">Store</a></div>
+        <div><a href="#">Mac</a></div>
+        <div><a href="#">iPod</a></div>
+        <div><a href="#">iPhone</a></div>
+        <div><a href="#">iPad</a></div>
+        <div><a href="#">iTunes</a></div>
+        <div><a href="#">Support</a></div>
+        <div class="search-container"><input type="text" placeholder='&#xf002;'
+            style="font-family: FontAwesome;color:white;" /></div>
+      </div>
+      <div>
+        <section>
+          <h1>What will your verse be ?</h1>
+          <p> Each of us has something to share.see how people are using <br />ipad in the unique ways to add their
+            worlds's
+            story
+          </p>
+          <p>
+            <a href="#" class="play-button-container">
+              Watch the "Your verse" film and explore the stories <img src="./img/play.png" height="30" width="30"
+                alt="play button">
+            </a>
+          </p>
+        </section>
+      </div>
+
+    </div>
+    <div class="apple-image-container">
+      <div><img src="./img/Iphone1.jpg" alt="iphone image" /></div>
+      <div><img src="./img/Iphone2.jpg" alt="iphone image" /></div>
+      <div><img src="./img/Iphone3.jpg" alt="iphone image" /></div>
+      <div><img src="./img/Iphone4.jpg" alt="iphone image" /></div>
+
+
```

Comment:
[REQUIRED] These four articles should have links as in the original site.[orignal page](https://web.archive.org/web/20140301004610/http://www.apple.com/)
[REQUIRED] Let's fix the HTML validator errors. I am using [this validator](https://validator.w3.org/#validate_by_upload)

Reply:
Resolved